1.1/files/docker-entrypoint.sh

```diff
@@ -76,16 +76,41 @@ if [[ $GENERATE_NEW_SAVE == true ]]; then
     fi
 fi
 
+if [[ -z "${FACTORIO_SERVER_ID_LOCATION:-}" ]]; then
```

Comment:
To set default values you could change this if block to

```bash
: ${FACTORIO_SERVER_ID_LOCATION:=/factorio/config/server-id.json}
```

Example:
```
: ${test:=1 2} && echo $test
1 2

: ${test:=3 4} && echo $test
1 2
```
The `:` is to make the command a nop, but the default value still gets set